commit d72baba6f53570d9719dd1be19538f2584af8e99
Author: Alex Hung <alex.hung@canonical.com>
Date:   Fri Mar 17 00:48:32 2017 +0000

    ACPI / blacklist: add _REV quirks for Dell Precision 5520 and 3520
    
    [ Upstream commit 9523b9bf6dceef6b0215e90b2348cd646597f796 ]
    
    Precision 5520 and 3520 either hang at login and during suspend or reboot.
    
    It turns out that that adding them to acpi_rev_dmi_table[] helps to work
    around those issues.
    
    Signed-off-by: Alex Hung <alex.hung@canonical.com>
    [ rjw: Changelog ]
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
    
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linux-4.9.47/drivers/acpi/blacklist.c
===================================================================
--- linux-4.9.47.orig/drivers/acpi/blacklist.c
+++ linux-4.9.47/drivers/acpi/blacklist.c
@@ -188,6 +188,22 @@ static struct dmi_system_id acpi_rev_dmi
 		      DMI_MATCH(DMI_PRODUCT_NAME, "Latitude 3350"),
 		},
 	},
+	{
+	 .callback = dmi_enable_rev_override,
+	 .ident = "DELL Precision 5520",
+	 .matches = {
+		      DMI_MATCH(DMI_SYS_VENDOR, "Dell Inc."),
+		      DMI_MATCH(DMI_PRODUCT_NAME, "Precision 5520"),
+		},
+	},
+	{
+	 .callback = dmi_enable_rev_override,
+	 .ident = "DELL Precision 3520",
+	 .matches = {
+		      DMI_MATCH(DMI_SYS_VENDOR, "Dell Inc."),
+		      DMI_MATCH(DMI_PRODUCT_NAME, "Precision 3520"),
+		},
+	},
 #endif
 	{}
 };
Index: linux-4.9.47/dummy/rpi_1447_d72baba6f53570d9719dd1be19538f2584af8e99.txt
===================================================================
--- /dev/null
+++ linux-4.9.47/dummy/rpi_1447_d72baba6f53570d9719dd1be19538f2584af8e99.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
